src/test/scala/is/hail/annotations/UnsafeSuite.scala

```diff
@@ -25,26 +26,30 @@ class UnsafeSuite extends SparkSuite {
       .filter { case (t, a) => a != null }
     val p = Prop.forAll(g) { case (t, a) =>
       t.typeCheck(a)
-      val f = t.fundamentalType
 
-      region.clear()
-      rvb.start(f)
-      rvb.addRow(t, a.asInstanceOf[Row])
-      val offset = rvb.end()
-      val ur = new UnsafeRow(t, region, offset)
+      println("loop")
+      CodecSpec.codecSpecs.foreach { codecSpec =>
+        println(JsonMethods.compact(JsonMethods.render(codecSpec.toJSON)))
```

Comment:
another one, though I understand why it's here. We should really have something like forAll but actually for every single element of this seq.